hotel_daily_service/wizard/hotel_daily_service_wizard.py

```diff
@@ -0,0 +1,62 @@
+from datetime import timedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class HotelDailyServiceLineWizard(models.TransientModel):
+    _name = "hotel.daily.service.line.wizard"
+    _description = "Hotel Daily Service Line"
+
+    @api.model
+    def _default_hotel_service_id(self):
+        rec = self.env["hotel.services"].search([], limit=1)
+        if not rec:
+            raise ValidationError(_("Please define a service first."))
+        return rec
+
+    hotel_folio_id = fields.Many2one(
+        comodel_name="hotel.folio", string="Folio"
+    )
+    hotel_service_id = fields.Many2one(
+        comodel_name="hotel.services",
+        string="Service",
+        default=_default_hotel_service_id,
+    )
+    quantity = fields.Integer(string="Quantity", default=1)
+    include_checkin_day = fields.Boolean(
+        string="Include Arrival Day", default=False
```

Comment:
Can you add a help here, will be good to identify the purpose of the field and its impact.

Reply:
Done!